apollo-gradle-plugin/src/test/testProject/android/buildScriptFixtures/flavored.gradle

```diff
@@ -48,6 +55,8 @@ repositories {
 
 dependencies {
   compile 'com.android.support:appcompat-v7:25.1.1'
+  compile 'com.google.code.findbugs:jsr305:3.0.1'
```

Comment:
The downside to this approach, is that we'd have to add the apollo-api dependencies manually.

I'm not sure if there's a way to avoid having to do that.